### PR TITLE
Scrape take 2

### PR DESCRIPTION
**Pull request by luuuk**
Ignore BizObject and Places - those are still WIP!

### CODE REVIEW

File: src/bizObj.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": 69420,
+    "name": "Fake Business",
+    "address": "69 420 Street, Seattle, WA, 98115",
+    "url": "https://www.eatmyass.com",
```
Comment (mackenzieleach) [inline src/bizObj.json:5]: 10/10 really fits the app theme

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+YELP_API_KEY=6Ialhkd_qwKhYNYw38mRefedsZqtjrJFYf_rRZsmSE-7iDVEtuez6PYif6MRUhz5ZEGVi7ffnExj9JObNmGEtqCRutd8xfdZAad3YUcAmiBcnmIUiIiOsOYgUaB8YHYx
```
Comment (mackenzieleach) [inline .env:1]: You may want to look into using Github Secrets to obscure your public API key on Github

**Review by mackenzieleach — APPROVED** [review]
I'm super excited you got so much done on the scraper!

**Comment by luuuk** [comment]
Merging Scraping skeleton code to main

File: .env (hunk 2 above)
Comment (luuuk) [reply .env:1]: This is actually a convo we should have w Wilson - what's the best way to grant API access to all users of the app? Should I use my API key (and hide it w gh secrets), or have users get their own keys?

**Pull request merged**